data/moves.ts

```diff
@@ -9782,12 +9782,12 @@ export const Moves: {[moveid: string]: MoveData} = {
 			const lastMove = target.lastMove;
 			const moveIndex = target.moves.indexOf(lastMove.id);
 			const noInstruct = [
-				'assist', 'beakblast', 'belch', 'bide', 'celebrate', 'copycat', 'dynamaxcannon', 'focuspunch', 'iceball', 'instruct', 'kingsshield', 'mefirst', 'metronome', 'mimic', 'mirrormove', 'naturepower', 'obstruct', 'outrage', 'petaldance', 'rollout', 'shelltrap', 'sketch', 'sleeptalk', 'struggle', 'thrash', 'transform', 'uproar',
+				'assist', 'beakblast', 'belch', 'bide', 'blazingtorque', 'bounce', 'celebrate', 'chatter', 'combattorque', 'copycat', 'dig', 'dive', 'dynamaxcannon', 'fly', 'focuspunch', 'freezeshock', 'geomancy', 'holdhands', 'iceball', 'iceburn', 'instruct', 'kingsshield', 'magicaltorque', 'mefirst', 'meteorassault', 'metronome', 'mimic', 'mirrormove', 'naturepower', 'noxioustorque', 'obstruct', 'outrage', 'petaldance', 'phantomforce', 'razorwind', 'rollout', 'shadowforce', 'shelltrap', 'sketch', 'skullbash', 'skyattack', 'skydrop', 'sleeptalk', 'solarbeam', 'solarblade', 'struggle', 'thrash', 'transform', 'uproar', 'wickedtorque',
```

Comment:
I saw a few charge and recharge moves that didn't have the NoInstruct flag, but since Hyper Beam apparently can't be used by instruct but it doesn't have the NoInstruct flag, it seems the game has a separate check aside from the flag. There seem to be some moves that would fail if run through both though, for some reason.